deployment/v3/mosip/config-server/delete.sh

```diff
@@ -12,7 +12,7 @@ while true; do
     if [ $yn = "Y" ]
       then
         kubectl -n $NS delete configmap global keycloak-host activemq-activemq-artemis-share s3 email-gateway
-        kubectl -n $NS delete secret db-common-secrets keycloak keycloak-client-secrets activemq-activemq-artemis softhsm-kernel softhsm-ida s3 email-gateway prereg-captcha
+        kubectl -n $NS delete secret db-mosip-audit-secret db-mosip-authdevice-secret db-mosip-credential-secret db-mosip-digitalcard-secret db-mosip-hotlist-secret db-mosip-ida-secret db-mosip-idmap-secret db-mosip-idp-secret db-mosip-idrepo-secret db-mosip-kernel-secret db-mosip-keymgr-secret db-mosip-master-secret db-mosip-pms-secret db-mosip-prereg-secret db-mosip-regdevice-secret db-mosip-regprc-secret db-mosip-resident-secret db-mosip-toolkit-secret keycloak keycloak-client-secrets activemq-activemq-artemis softhsm-kernel softhsm-ida s3 email-gateway prereg-captcha
```

Comment:
Can a regex search be used here like the one used in copy_secrets.sh file? This will be hard to maintain later on when more secrets are added

Reply:
yes we can do that changes and add that in delete.sh script.